blazor/datagrid/columns.md

```diff
@@ -379,6 +379,77 @@ The following image represents foreign key column
 > * For remote data, the sorting and grouping is done based on [ForeignKeyField](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ForeignKeyField) instead of [ForeignKeyValue](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ForeignKeyValue).
 > * If [ForeignKeyField](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ForeignKeyField) is not defined, then the column uses [Field](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_StackingGroup) property of **GridColumn** tag helper.
 
+### Prevent filter query generation for foreignkey column
+
+By default, a filter query for the foreignkey column will be generated based on the foreignkey value. When the URL query string is too long (containing the filter queries), exception occurs while performing filtering operations. To overcome this issue, we can prevent filter query generation for the foreignkey column. This can be achieved by setting the PreventFilterQuery argument of the OnActionBegin event to true.
```

Comment:
don't mention like exception occurs like that. need to remove second point in this para. Just mention like have an option to customize the filter query that is enough